### ISSUE TITLE
Improve reading of tables in microsoft word.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2278]
**Reported by ashleycox on 2012-04-27 10:06**
Currently, nvda can read tables in word if they have content in them. However, if the table has no content, or a column is empty, NVDA simply doesn't announce that columns existence. This makes it difficult to insert data into tables. NVDA needs to announce columns, regardless of the content in them. Using word 2007.

**Comment by leonardder** [comment]
See [Table with empty columns and rows.docx](https://github.com/nvaccess/nvda/files/1150176/Table.with.empty.columns.and.rows.docx)

Strange enough, this table was very buggy in terms of table navigation when I kopied it from Excel to Word 2016, but when I restarted Word, all seemed fixed.

**Comment by bdorer** [comment]
Using Word 2013 and latest nvda-alpha everything is working for me. I wote for closing as worksforme. If someone is able to provide steps to reproduce we can reopen.

**Comment by leonardder** [comment]
Closing, as the report is from Word 2007 which is over 12 years old now.
